src/backend/distributed/connection/connection_management.c

```diff
@@ -725,6 +731,8 @@ AfterXactHostConnectionHandling(ConnectionHashEntry *entry, bool isCommit)
 		MultiConnection *connection =
 			dlist_container(MultiConnection, connectionNode, iter.cur);
 
+		UnsetCitusNoticeLevel();
```

Comment:
This seems like a weird place to put this. It gets called for every connection and seems unrelated to closing connections. Should it be in the abort handler?

Reply:
(`CoordinatedTransactionCallback`)